cmake/VowpalWabbitUtils.cmake

```diff
@@ -2,7 +2,9 @@ include(CMakeParseArguments)
 include(GNUInstallDirs)
 include(VWFlags)
 include(DetectCXXStandard)
-set(VW_CXX_STANDARD 11)
+if(NOT VW_CXX_STANDARD)
```

Comment:
Curious.  In all the CI's will it be C++ 11 by default?

Reply:
Yes, unless otherwise specified 11 is the default.